src/Dofs/DofHandler.jl

```diff
@@ -718,9 +705,6 @@ For more details we refer to [1] as we follow the methodology described therein.
     !!!TODO Investigate if we can somehow pass the interpolation into this function in a typestable way.
 """
 @inline function permute_and_push!(cell_dofs::Vector{Int}, dofs::StepRange{Int,Int}, orientation::SurfaceOrientationInfo, adjust_during_distribution::Bool)
-    if adjust_during_distribution && length(dofs) > 1
-        error("Dof distribution for interpolations with multiple dofs per face not implemented yet.")
```

Comment:
Is this check moved elsewhere?

Reply:
Good spot! Not AFAIK. 
@lijas do you know more?

Reply:
Fixed by checking reference dim of cell. This should be safe in all cases except two cells on top of eachother, which is the same limitation as the check on master. Diff showing fix: https://github.com/Ferrite-FEM/Ferrite.jl/pull/914/commits/5223729c4a0b0683e331f0535602d0125c3cb52c